docs/gen-ai/openai.md

```diff
@@ -11,7 +11,7 @@ linkTitle: OpenAI traces and metrics
 <!-- toc -->
 
 - [OpenAI Span attributes](#openai-span-attributes)
-- [OpenAI Metric attributes](#openai-metric-attributes)
+- [OpenAI Metrics](#openai-metrics)
```

Comment:
Minor: Why the difference in titling between span and metrics? One includes the word `attributes`, the other does not?

Reply:
good catch!